src/pages/index.js

```diff
@@ -83,6 +66,7 @@ function generateCard(item) {
 function renderCard(data) {
   const card = new Card(data, "#card-template", {
     handleImageClick: () => imagePopup.open(data),
+    handleDeleteClick,
   });
   return card;
```

Comment:
`.getCard()` should be inside `renderCard` not to duplicate it if you decide to add cards with `append`

Reply:
* You haven't corrected my remark https://github.com/MrSurge87/se_project_aroundtheus/pull/1#discussion_r1387589705